sonar-project.properties

```
@@ -0,0 +1,21 @@
+sonar.organization=sap-1
+sonar.projectKey=SAP_open-ux-tools
+sonar.sources=.
+sonar.inclusions=**/*.ts
+sonar.exclusions=**/*.test.ts
```

```suggestion
sonar.exclusions=**/*.test.ts, **/templates/**
```
I think the templates should be excluded. Is this the place to configure it?

---

@devinea: Yes, I added this in the latest commit.